### DISCUSSION TITLE
reuse client instance or not?

### DISCUSSION THREAD

**Discussion by alex-jitbit** [MichaCo/DnsClient.NET#130] [Q&A] [answered]
I was wondering if it is the best practice - to reuse the client instance (or is it ok to recreate every time)

**Comment by alex-jitbit** [comment, marked as answer]
Nevermind, found it in the docs, sorry!

**Reply by martinguenther** [reply]
I'm interested in this too, but could not find the answer in the docs.
Could you please point out what you found or where you found it? 
THX

**Reply by martinguenther** [reply]
I was obviously blind... It's there, right on the homepage, in the "Usage" section. I was looking at /docs. Sorry!

> It is important to note, that it is highly recommended to instantiate the lookup client only once per application and to always use the same reference (singleton).
